### ISSUE TITLE
separate development ontology from release ontology

### ISSUE THREAD

**Issue by wdduncan** [iofoundry/Core#1]
In many OBO ontologies, such the [Environment Ontology (envo)](https://github.com/EnvironmentOntology/envo) and [Environmental Exposure Ontology (ecto)](https://github.com/EnvironmentOntology/environmental-exposure-ontology), we have found it extremely helpful to develop the ontology in a `src/ontology` directory and then use github actions perform error checking on change to the development file. Once the development file is ready new release can be made to the top level.

For example, here the simplified view of ecto's directory structure:
```
root
  - ecto.owl
  - src
      - ontology
          - ecto-edit.owl
```
The ontology editors make changes, using github branches and pull requests, to `src/ontology/ecto-edit.owl`. When a new pull request (i.e., change) is merged, we perform quality checks, and when ready, [robot](http://robot.obolibrary.org/) is used to build the final release product `ecto.owl`. This also allows us to use functional syntax, which is much more readable, for the `ecto-edit.owl` file and then have the release project use rdf/xml.

The advantage of this is that lots of activity can be going on concerning the edit file, but the release at the top level always remains stable. 

There are  lots of details to the workflow that happy to discuss.
